docs/source/advanced/profiler.rst

```diff
@@ -204,3 +204,47 @@ Use::
 Or::
 
     python -c 'import torch; print(torch.autograd.profiler.load_nvprof("trace_name.prof"))'
+
+
+XLA Profiling
+-------------
+
+:class:`~pytorch_lightning.profiler.xla.XLAProfiler` will help you debug and optimize training
+workload performance for your models using Cloud TPU performance tools.
+
+.. code-block:: python
+
+    # by passing the `XLAProfiler` alias
+    trainer = Trainer(..., profiler="xla")
+
+    # or by passing an instance
+    from pytorch_lightning.profiler import XLAProfiler
+
+    profiler = XLAProfiler(port=9012)
+    trainer = Trainer(..., profiler=profiler)
+
+
+Manual capture via TensorBoard
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
@rohitgr7 started with standardizing the titles, let's make it capitalized :)) sorry for the nit ^^